package.json

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "cli",
+	"version": "0.0.0",
+	"license": "MIT",
+	"bin": {
+		"asyncapi": "dist/cli.js",
+		"@asyncapi/cli": "dist/cli.js"
+	},
+	"engines": {
+		"node": ">=10"
+	},
+	"scripts": {
+		"build": "tsc",
+		"start": "npm run build && node dist/cli.js",
+		"pretest": "npm run build",
+		"pretest:coverage": "npm run build",
+		"test": "jest",
+		"test:coverage": "jest --verbose --coverage"
```

Comment:
we need scripts `lint` and `generate:assets` to satisfy automation workflows. For `generate:assets` you can just echo that for now nothing is generated

Reply:
for linting, do you have a specific linter and rules?? 🤔

Reply:
@jotamusik not for TS, not yet, but we will probably adopt what we do at https://github.com/asyncapi/modelina

Reply:
Should I add it now, or create an for this?? 🤔

Reply:
we do not need linter now, but we do need a script, it can just echo `not implemented yet`

Reply:
done!